compiler/damlc/lib/DA/Cli/Damlc/Test.hs

```diff
@@ -103,11 +154,12 @@ failedTestOutput h file = do
 
 
 printTestCoverage ::
-       ShowCoverage
-    -> [LF.Module]
+    ShowCoverage
+    -> [LF.ExternalPackage]
+    -> [(Maybe LF.PackageId, LF.Module)]
     -> [(VirtualResource, Either SSC.Error SS.ScenarioResult)]
     -> IO ()
-printTestCoverage ShowCoverage {getShowCoverage} dalfs results
+printTestCoverage ShowCoverage {getShowCoverage} extPkgs dalfs results
```

Comment:
same name confusion betwene modules and dalfs also exists here (and did before this PR as well).